src/main/scala/rps/Game.scala

```diff
@@ -0,0 +1,43 @@
+package rps
+import java.util.Scanner
+import java.util.InputMismatchException
+
+object Game {
+  def moveRepr(move: Int) : String = {
+    move match {
+      case 0 => return "Rock"
+      case 1 => return "Paper"
+      case 2 => return "Scissors"
+      case _ => return "Invalid move!"
+    }
+  }
+
+  def detectGameEndResult(playerMove: Int, computerMove: Int) : String = {
+    (playerMove, computerMove) match {
+      case (x, y) if x == y => return "Draw!"
+      case (0, 1) => return "I Won!"
+      case (0, 2) => return "You Won!"
+      case (1, 0) => return "You won!"
```

Comment:
you can match multiple cases using `|`, e.g.

```scala
case (0, 2) | (1, 0) | (2, 1) => "You Won!"
```

This has limitations when you are binding variables, but it's fine in this case.